core/src/test/java/org/infinispan/util/AbstractControlledRpcManager.java

```diff
@@ -35,112 +34,20 @@ public AbstractControlledRpcManager(RpcManager realOne) {
-      log.trace("ControlledRpcManager.broadcastRpcCommandInFuture2");
-      beforeInvokeRemotely(rpcCommand);
-      realOne.broadcastRpcCommandInFuture(rpcCommand, usePriorityQueue, future);
-      afterInvokeRemotely(rpcCommand, null);
-   }
-
-   @Override
-   public void invokeRemotelyInFuture(Collection<Address> recipients, ReplicableCommand rpcCommand, NotifyingNotifiableFuture<Object> future) {
-      log.trace("ControlledRpcManager.invokeRemotelyInFuture1");
-      beforeInvokeRemotely(rpcCommand);
-      realOne.invokeRemotelyInFuture(recipients, rpcCommand, future);
-      afterInvokeRemotely(rpcCommand, null);
-   }
-
-   @Override
-   public void invokeRemotelyInFuture(Collection<Address> recipients, ReplicableCommand rpcCommand, boolean usePriorityQueue, NotifyingNotifiableFuture<Object> future) {
-      log.trace("ControlledRpcManager.invokeRemotelyInFuture2");
-      beforeInvokeRemotely(rpcCommand);
-      realOne.invokeRemotelyInFuture(recipients, rpcCommand, usePriorityQueue, future);
-      afterInvokeRemotely(rpcCommand, null);
-   }
-
-   @Override
-   public void invokeRemotelyInFuture(Collection<Address> recipients, ReplicableCommand rpcCommand, boolean usePriorityQueue, NotifyingNotifiableFuture<Object> future, long timeout) {
-      log.trace("ControlledRpcManager.invokeRemotelyInFuture3");
-      beforeInvokeRemotely(rpcCommand);
-      realOne.invokeRemotelyInFuture(recipients, rpcCommand, usePriorityQueue, future, timeout);
-      afterInvokeRemotely(rpcCommand, null);
-   }
-
-   @Override
-   public void invokeRemotelyInFuture(Collection<Address> recipients, ReplicableCommand rpcCommand, boolean usePriorityQueue, NotifyingNotifiableFuture<Object> future, long timeout, boolean ignoreLeavers) {
-      log.trace("ControlledRpcManager.invokeRemotelyInFuture4");
-      beforeInvokeRemotely(rpcCommand);
-      realOne.invokeRemotelyInFuture(recipients, rpcCommand, usePriorityQueue, future, timeout, ignoreLeavers);
-      afterInvokeRemotely(rpcCommand, null);
+   public CompletableFuture<Map<Address, Response>> invokeRemotelyAsync(Collection<Address> recipients,
+                                                                        ReplicableCommand rpc,
+                                                                        RpcOptions options) {
+      log.trace("ControlledRpcManager.invokeRemotelyAsync");
+      beforeInvokeRemotely(rpc);
+      CompletableFuture<Map<Address, Response>> future = realOne.invokeRemotelyAsync(recipients, rpc,
+            options);
+      afterInvokeRemotely(rpc, null);
+      return future;
    }
 
    @Override
    public Map<Address, Response> invokeRemotely(Collection<Address> recipients, ReplicableCommand rpc, RpcOptions options) {
-      log.trace("ControlledRpcManager.invokeRemotely6");
+      log.trace("ControlledRpcManager.invokeRemotely");
```

Comment:
I believe the intent was for all of these to have unique numbers for easier debugging in testing.  Seems we didn't do that properly on some.  Do we want to keep trying that?